install/sync_logfiles.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 
-DESTINATION_PATH="example/path"
-SERVER="hostname_or_ip"
-USER="Posseidon"
-FILES_LOCATION="~/Poseidon/www/webroot/record"
-PASSWORD_LESS="poseidon"
-KEEP_ORIGINAL_FILES=true
+
+DESTINATION_PATH="destination_Path"
+SERVER="hostname or ip"
+USER="HB_XX"
+FILES_LOCATION="/home/ubuntu/Poseidon/www/webroot/record/" # without the last '/' it will transfer the folder instead of files
+PASSWORD_LESS="test"
```

Comment:
Why _LESS ?

Reply:
could change the name of the variable to rsa_key